include/OpenSpaceToolkit/Astrodynamics/Dynamics/CentralBodyGravity.hpp

```diff
@@ -25,90 +25,79 @@ using ostk::physics::time::Instant;
 
 using ostk::astro::Dynamics;
 
-/// @brief                      Define the acceleration experienced by a point mass due to gravity
-
+/// @brief     Define the acceleration experienced by a point mass due to gravity
 class CentralBodyGravity : public Dynamics
 {
    public:
-    /// @brief                  Constructor
+    /// @brief Constructor
     ///
-    /// @code
-    ///                         const aCelestial = { ... };
-    ///                         CentralBodyGravity centralBodyGravity = { aCelestial };
+    /// @code{.cpp}
+    ///                  const aCelestial = { ... };
+    ///                  CentralBodyGravity centralBodyGravity = { aCelestial };
     /// @endcode
     ///
-    /// @param                  [in] aCelestial A Celestial Object
-
+    /// @param aCelestial A Celestial Object
     CentralBodyGravity(const Shared<const Celestial>& aCelestial);
 
-    /// @brief                  Constructor
+    /// @brief Constructor
     ///
-    /// @code
-    ///                         const aCelestial = { ... };
-    ///                         const aName = { ... };
-    ///                         CentralBodyGravity centralBodyGravity = { aCelestial, aName };
+    /// @code{.cpp}
```

Comment:
Should we make the @code systematic? It seems it's applied to come constructor only (can be done separately)

Reply:
Yeah I think adding more documentation should be done in a separate MR.